src/ResourceManagement/WebSite/WebSite.Tests/ScenarioTests/WebSite.ScenarioTests.cs

```diff
@@ -218,7 +218,7 @@ public void GetAndSetSensitiveSiteConfigs()
                     var connStringValueTypePair = new ConnStringValueTypePair
                     {
                         Value = connectionStringValue,
-                        Type = DatabaseServerType.MySql
```

Comment:
>DatabaseServerType [](start = 31, length = 18)

I don't see any new test for the new functionality that was added. Can you add test coverage for the new functionality.